engine/graphics/src/test/test_app_graphics_package_assets.py

```diff
@@ -51,6 +60,10 @@ def write_header(header_path, assets):
 
 if __name__ == '__main__':
 	write_header(
-		"test_app_vulkan_assets.h",
-		[get_buffer_str("vertex_program", "test_app_vulkan.vs", "vert"),
-		 get_buffer_str("fragment_program", "test_app_vulkan.fs", "frag")])
\ No newline at end of file
+		"test_app_graphics_assets.h",
+		[to_glsl("glsl_vertex_program", "test_app_graphics.vs"),
+		 to_glsl("glsl_fragment_program", "test_app_graphics.fs"),
+		 to_glsl("glsl_compute_program", "test_app_graphics.compute"),
+		 to_spirv("spirv_vertex_program", "test_app_graphics.vs", "vert"),
+		 to_spirv("spirv_fragment_program", "test_app_graphics.fs", "frag"),
+		 to_spirv("spirv_compute_program", "test_app_graphics.compute", "compute")])
```

Comment:
Are these assets cross platform?
Can I run them on Nintendo Switch?

Reply:
Yes, all of these will be compiled into .spv counterparts, and they should be able to run on any vulkan driver.